posthog/api/survey.py

```diff
@@ -255,9 +259,20 @@ def validate_questions(self, value):
             if not isinstance(raw_question, dict):
                 raise serializers.ValidationError("Questions must be a list of objects")
 
-            cleaned_question = {
-                **raw_question,
-            }
+            # Create a copy of the question without stable_index
+            # This ensures users can't manually set stable_index through the API
```

Comment:
if this is a new field that will be used in the SDK, and part of the API spec, we should allow the API to set this